docs/UserGuide.md

```diff
@@ -58,29 +67,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
+### Viewing help: `help` 
 
-![help message](images/helpMessage.png)
+Shows a message explaining how to access the help page.
 
 Format: `help`
 
+### Adding a client: `client add` 
 
-### Adding a person: `add`
+Adds a new client to the app.
 
-Adds a person to the address book.
+Format: `client add n/NAME p/PHONE e/EMAIL a/ADDRESS c/COUNTRY tz/TIMEZONE`
```

Comment:
Maybe we should change PHONE to PHONE_NUMBER, following the original format, for clarity